src/reference/docbook/whats-new.xml

```diff
@@ -108,5 +108,25 @@
 				gateway; the dependency was removed for those users who don't need it.
 			</para>
 		</section>
+		<section id="4.2-jms-changes">
+			<title>JMS Changes</title>
+			<para>
+				Previously, if a JMS message-driven inbound channel adapter or inbound gateway could
+				not deliver a message due to a conversion error, an exception would be thrown back to the
+				container. If the container was configured to use transactions, the message would be
+				rolled back and redelivered repeatedly. The conversion process occurs before and during
+				message construction so such errors were not sent to the error channel.
+			</para>
+			<para>
+				With this release, such conversion exceptions now result in an <classname>ErrorMessage</classname>
+				being sent to the error channel, with the exception as the payload. If you wish the
+				transaction to be rolled back, and you have an error channel defined, the integration
+				flow on the error channel must re-throw the exception (or another). If the error flow
+				does not throw an exception, the transaction will be committed and the message removed.
+			</para>
+			<para>
+				If no error-channel is defined, the exception is thrown back to the container, as before.
+			</para>
```

Comment:
Actually nothing to complain about, but this text looks like a permanent candidate to the JMS chapter.
Will be done on merge.